src/Pages/Recommend/RecommendComponent/RecommendList.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import RecommendListBox from "../RecommendComponent/RecommendListBox";
+import styled from "styled-components";
+
+const API = "http://10.58.1.184:8000/recommend?type=written";
+
+export class RecommendList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userList: [],
+      open: false
+    };
+  }
+
+  checkModalHandler = e => {
+    this.setState({
+      open: !this.state.open
+    });
+  };
+
+  handleDeleteItem = id => {
+    const filterArray = this.state.userList.filter(user => {
+      if (user.id !== id) {
+        return user;
+      }
+    });
```

Comment:
filter 메소드는 return true 일 때 그 요소를 반환하고 false일 때 반환하지 않기 때문에 user를 return 하지 않으셔도 됩니다!
```suggestion
    const filterArray = this.state.userList.filter(user => user.id !== id}
    });
```